### PR TITLE
CTW-327 Display conditions grouping ccs

### PR DESCRIPTION
**Pull request by melocule**
Conditions table displays CCS groupings when available, in a new column beside condition names. Some condition names are detailed to the point of reduced readability, and CCS groupings provide a more readable description of the condition.
image

### CODE REVIEW

File: src/models/conditions.ts [hunk 1]
```diff
@@ -83,6 +83,10 @@ export class ConditionModel {
     return findCoding(SYSTEM_ICD10, this.resource.code)?.display;
   }
 
+  get group(): string | undefined {
+    return findCoding(SYSTEM_CCS, this.resource.code)?.display;
+  }
+
```
Comment (luthfib) [inline src/models/conditions.ts:89]: Not sure how I feel about this name. I think it is too vague. I think something along the lines of `ccsGrouping` or `ccsCode`, or something a bit more descriptive as we have done when getting other codes.

**Comment by melocule** [comment]
FYI: Currently checking in on the design of the table to address differences between the Miro and the ticket description (whether ICD-10 codes should be displayed)

**Review by luthfib — APPROVED** [review]
LGTM

**Review by pmadria3 — APPROVED** [review]
LGTM

File: src/components/content/conditions-table-base.tsx [hunk 2]
```diff
@@ -22,15 +22,20 @@ export function ConditionsTableBase({
...
       title: "Status",
       dataIndex: "clinicalStatus",
-      className: "ctw-w-[20%]",
+      className: "ctw-w-[10%]",
     },
     {
       title: "Recorded Date",
       dataIndex: "recordedDate",
-      className: "ctw-w-[20%]",
+      className: "ctw-w-[10%]",
```
Comment (WesleyKapow) [inline src/components/content/conditions-table-base.tsx:38]: Currently adds to `110%` due to the `action-column`!!

**Pull request merged**